src/components/user/Asset.js

```diff
@@ -478,8 +491,8 @@ const TitleBodyWrapper = styled(BodyWrapper)`
 `;
 
 const TitleWrapper = styled.div`
-  margin-left: 58px;
-  margin-right: 50px;
+  margin-left: 40px;
+  margin-right: 40px;
```

Comment:
```suggestion
  margin: 0px 40px 0px 40px;
```
이렇게도 작성할 수 있을 거 같습니다 !